### PR TITLE
Change save exporting to give you a file to download

### PR DESCRIPTION
**Pull request by thePalindrome**
Usual stuff from me applies, this isn't compiled, sorry :/

Tested works perfectly fine for me, but might also want to check chrome just in case.

### CODE REVIEW

File: Technomancy/app.ts [hunk 1]
```diff
@@ -2869,6 +2869,10 @@ function load() {
 }
 
 function save_to_clip() { /* Put save data in clipboard. Copied from Stack Overflow :) */
+    alert("STUB!")
```
Comment (fuzzything44) [inline Technomancy/app.ts]: Any reason this needs to have functionality removed?

File: Technomancy/app.ts [hunk 2]
```diff
@@ -2877,34 +2881,16 @@ function save_to_clip() { /* Put save data in clipboard. Copied from Stack Overf
...
     let text = b64EncodeUnicode(JSON.stringify(localStorage));
-    let textArea: any = document.createElement("textarea");
-
-    /* Styling to make sure it doesn't do much if the element gets rendered */
+    let a: any = document.createElement("a");
 
-    /* Place in top-left corner of screen regardless of scroll position. */
-    textArea.style.position = 'fixed'; textArea.style.top = 0; textArea.style.left = 0; textArea.style.width = '2em'; textArea.style.height = '2em';
+    textArea.href = 'data:text/plain;charset=utf-8,' + encodeURIComponent(text);
+    textArea.download = "Technomancy.sav";
```
Comment (fuzzything44) [inline Technomancy/app.ts]: You're using `a` instead of `textArea`, remember.

File: Technomancy/app.ts [hunk 3]
```diff
@@ -2877,34 +2881,16 @@ function save_to_clip() { /* Put save data in clipboard. Copied from Stack Overf
...
-        }
-    } catch (err) {
-        $("#settings").addClass("hidden");
-        $("#events_topbar").html("Save Export - Secondary");
-        $("#events_content").html("<textarea class='fgc bgc' style='width: 90%; height: 90%;' readonly>" + text + "</textarea>");
-        $("#events_content textarea").select();
-        $("#events").removeClass("hidden");
-        
-    }
     document.body.removeChild(textArea);
```
Comment (fuzzything44) [inline Technomancy/app.ts]: Another instance of `textArea`

**Review by fuzzything44 — CHANGES_REQUESTED** [review]

File: Technomancy/app.ts (hunk 2 above)
Comment (thePalindrome) [reply Technomancy/app.ts]: Gah, my bad. problem with me making changes in the .js, then just replicating in the .ts
Fixed.

File: Technomancy/app.ts (hunk 3 above)
Comment (thePalindrome) [reply Technomancy/app.ts]: Second verse same as the first.

File: Technomancy/app.ts (hunk 1 above)
Comment (thePalindrome) [reply Technomancy/app.ts]: Because "Yes".

Reverted because it makes more sense than stubbing it (I don't remember why I did it).

**Comment by thePalindrome** [comment]
Alright, patched the things (man, sleepy pali is not the best at coding, who'd've thought? :P )

**Comment by fuzzything44** [comment]
Looks good. Any bets on if I forget to mention it in the next changelog I write?

**Comment by thePalindrome** [comment]
Hmm, given this, I'll put it as a 20% chance you forget to mention it (it's moderately noticeable anyways)

Also, if you want to change around how it's laid out, feel free :+1:

**Pull request merged**